src/utils/RandomDataGenerator.js

```diff
@@ -0,0 +1,48 @@
+const { v4: uuidv4 } = require('uuid')
+
+class RandomDataGenerator {
+
+  static uuid() {
+    return uuidv4()
+  }
+
+  static boolean() {
+    return Math.random() <= 0.5
+  }
+
+  static integer(options = {}) {
+    const min = options.min || 0
+    const max = Number.isInteger(options.max) ? options.max : 1000
+    return Math.floor(Math.random() * (max - min + 1)) + min
+  }
```

Comment:
this will not work if somebody does min: -100, max: 0, because 0 is considered falsy. YOu might be better served with default arguments here, as they are online assigend when the key is not defined: 

```js
static integer({ min = 0, max = 1000 } = {})
```
You may want to check if min is smaller than max and throw an error is somebody messes that up

Reply:
i had it as static integer({ min = 0, max = 1000 }) but that blows up without opts, your solution does make way more sense

Reply:
as discussed, no error throwing in production code, worst case its gonna give a negative but the props are pretty clear